### DISCUSSION TITLE
Looking for a way to Add to ModuleActions

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#1311] [Q&A]
When in edit mode I would like to be able to add menu options to the ModuleActions menu from within my module.  Is this possible?

**Comment by sbwalker** [comment]
Oqtane does not currently have the equivalent of the DNN IActionable interface currently. It relies on the ActionLink or ActionDialog components instead. I would be curious from a UX perspective to hear your thoughts on why you would prefer to add custom ModuleActions rather than using ActionLink or ActionDialog.

**Reply by leigh-pointer** [reply]
When a module is not your general View Edit Settings format and there are a few more areas of the module to navigate, it can end up with quite a few buttons with long captions.
